examples/extensions/add_geo_target.rb

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example adds a geo target to an extension feed item for targeting.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def add_geo_target(customer_id, feed_item_id, geo_target_constant_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  if geo_target_constant_id.nil?
+    geo_target_constant_id = GEO_TARGET_CONSTANT_ID
+  end
+
+  resource_name = client.path.extension_feed_item(customer_id, feed_item_id)
+
+  # Creates the update operation for extension feed item using the
+  # specified feed item ID and geo target constant ID for targeting.
+  operation = client.operation.update_resource.extension_feed_item(resource_name) do |efi|
+    efi.targeted_geo_target_constant = client.path.geo_target_constant(geo_target_constant_id)
+  end
+
+  # Issues a mutate request to update the extension feed item.
+  response = client.service.extension_feed_item.mutate_extension_feed_items(
+    customer_id: customer_id,
+    operations: [operation]
+  )
+
+  # Prints the resource name of the updated extension feed item.
+  puts "Updated extension feed item with resource name: " \
+    "'#{response.results.first.resource_name}'"
+end
+
+if __FILE__ == $0
+  # Geo target constant ID for US.
+  GEO_TARGET_CONSTANT_ID = 2840
+
+  options = {}
```

Comment:
Add defaults up here to match other examples.

Reply:
Done